nautobot_golden_config/templates/nautobot_golden_config/content_template.html

```diff
@@ -81,6 +81,12 @@
                             <a href="{% url 'plugins:nautobot_golden_config:configcompliance_details' pk=device.pk config_type='intended' %}"><i class="mdi mdi-text-box-check-outline"></i></a>
                         </td>
                     </tr>
+                    <tr>
```

Comment:
This should not be considered `{% if config_features.intended and golden_config.intended_config %}` stanza, and and should be in it's own stanza based on the new configuration

Reply:
yes